community/joe/version

```diff
@@ -0,0 +1 @@
+4.6 4
```

Comment:
relver should stay at 1 and should only be bumped after it's in the repo and needs to be rebuilt

Reply:
I didn't know that sounds good.